INSTALL.md

```diff
@@ -347,12 +448,3 @@ libepetra.so          -> No prefix
 libtrilinos_epetra.so -> -DTRILINOS_PREFIX="trilinos_"
 ```
 
-## Applications
-
-Specific compilation information about applications can be found in their own directories:
-
-- [EigenSolvers Application](applications/EigenSolversApplication/README.md#build-instructions)
-- [HDF5 Application](applications/HDF5Application/README.md#build-instructions)
-- [MultilevelMontecarlo Application](applications/MultilevelMonteCarloApplication/README.md#external-libraries)
-- [Poromechanics Application](applications/PoromechanicsApplication/README.md#how-to-use-mpi-in-poromechanics-application)
-- [Trilinos Application (Aditional notes)](applications/TrilinosApplication/README.md#notes-for-compilation)
```

Comment:
@philbucher  there were already link to apps README's  😅 I have moved to the top and added a few